samples/v1.0/Tests/FactSet.Test.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Can we update the name of this file to describe what is being tested?

Reply:
sure, I will add a new card called CardsWithInvalidProperties.json that combines  TextBlock.Test.json and FactSet.Test.json, and remove the old cards. I will add the new card in a new PR.

---
In reply to: [494599248](https://github.com/microsoft/AdaptiveCards/pull/4809#discussion_r494599248) [](ancestors = 494599248)